### PR TITLE
Added existing dockerfiles used by che.

### PR DESCRIPTION
**Pull request by JamesDrummond**
1. Copied everything from eclipse/che repo BUT /dockerfiles/Che-server folder.  Removed the leading /dockerfiles folder after the move. Copied /dockerfiles/che-server/Dockerfile file to root directory.

2. Copied the contents of the codenvy/dockerfiles repo to a directory called /recipes.

### CODE REVIEW

**Comment by JamesDrummond** [comment]
Need to change the commiter email. used gmail instead of codenvy.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+USER user
+
+ADD assembly/assembly-main/target/eclipse-che-*/eclipse-che-* /home/user/che/
+
+ENV CHE_HOME /home/user/che
+
+ENTRYPOINT [ "/home/user/che/bin/che.sh", "-c" ]
+
+CMD [ "run" ]
```
Comment (TylerJewell) [inline Dockerfile]: This file should not be in this repository.  This file should remain in /che repository, but moved from /dockerfiles/che-server/Dockerfile to be just /Dockerfile.  We can then delete the /dockerfiles/che-server directory.

Also note - if you move this file to the root of the Che repo, then the comment on how to build this dockerfile needs to change.  The syntax is then wrong (has the wrong context.

**Comment by TylerJewell** [comment]
@eivantsov @benoitf @riuvshin - please take a look.  Assuming that we can still release and build images independently of Che, are you comfortable with the overall structure?

**Comment by TylerJewell** [comment]
@JamesDrummond - please verify that every file has appropriate license + copyright header in the opening comments.

**Pull request closed without merging**